inventory/sample/group_vars/k8s_cluster/addons.yml

```diff
@@ -104,6 +104,13 @@ ingress_publish_status_address: ""
 #     operator: "Equal"
 #     value: ""
 #     effect: "NoSchedule"
+# ingress_nginx_resources:
```

Comment:
In Kubespray, most components have the separated options for each values like
- flannel_cpu_limit
- flannel_memory_limit
- flannel_cpu_requests
-  flannel_memory_requests

as calico_node_cpu_limit, calico_node_memory_limit etc.
How about having the separated options for ingress_nginx for the consistency of Kubespray?

Reply:
Thanks for the hint, I agree with you it will be better, I changed my PR.